src/common/classes/init.cpp

```diff
@@ -102,7 +102,7 @@ namespace
 			if (dladdr((void*) &allClean, &path))
 			{
 				name = path.dli_fname;
-				name += ".memdebug.log";
```

Comment:
Use of function call instead redefined operator does not make code better. 
Rollback such changes too.